### PR TITLE
small ux tweak

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/19094

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 1]
```diff
@@ -63,10 +63,11 @@
         <p>
           {{ $t('texts.beneficialOwnershipAssessmentText1') }}
         </p>
-        <br>
+        <!-- NB: might be added back in later -->
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:66]: if chance of reading is not high, I would remove it altogether :)

**Review by hfekete — APPROVED** [review]

File: btr-web/btr-main-app/components/individual-person/AddNew.vue (hunk 1 above)
Comment (kialj876) [reply btr-web/btr-main-app/components/individual-person/AddNew.vue:66]: yeah sounds good

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-97-lrgbhcng.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-97-lrgbhcng.web.app

**Pull request merged**